validation.rst

```diff
@@ -730,6 +730,27 @@ constraint that's applied to the class itself. When that class is validated,
 methods specified by that constraint are simply executed so that each can
 provide more custom validation.
 
+Validating Object With Inheritance
+----------------------------------
+
+When you validate an object that extends another class, the validator
+automatically validates constraints defined in the parent class as well.
+
+.. caution::
+
+    Note that overriding a property with others constraints in a child class
+    will not remove the constraints defined in the parent class on that same
+    property.
+    Instead, the constraints will be merged for that property.
+    This is related to Java Language Specification.
```

Comment:
maybe we can get rid of the caution box and write just regular text? What do you think?

The tip below is ok from my side 👍

Reply:
It was my first intention, but it may deserve to be more highlighted given the potential issues.
Let's wait for other opinions?